include/Nazara/Utility/Algorithm.hpp

```diff
@@ -66,36 +65,16 @@ namespace Nz
 	NAZARA_UTILITY_API void SkinPositionNormalTangent(const SkinningData& data, unsigned int startVertex, unsigned int vertexCount);
 
 	NAZARA_UTILITY_API void TransformVertices(VertexPointers vertexPointers, unsigned int vertexCount, const Matrix4f& matrix);
-	
+
 	template<typename T>
 	struct IsSuitableForComponent
```

Comment:
This should be moved to the .inl too (as it has no use for users) in a Detail namespace